src/CMakeLists.txt

```diff
@@ -43,7 +43,13 @@ add_library(driver_osvr
 	ServerDriver_OSVR.h
 	ValveStrCpy.h
 )
+
+if (WIN32)
+target_link_libraries(driver_osvr PRIVATE dxgi osvr::osvrClientKitCpp eigen-headers util-headers jsoncpp_lib)
+else()
 target_link_libraries(driver_osvr PRIVATE osvr::osvrClientKitCpp eigen-headers util-headers jsoncpp_lib)
+endif()
```

Comment:
You can leave the original `target_link_libraries()` line as is and then write
```
if(WIN32)
    target_link_libraries(driver_osvr PRIVATE dxgi)
endif()
```
and CMake will append the `dxgi` library to the list of dependencies.